templates/app/controllers/user_confirmations_controller.rb

```diff
@@ -1,17 +1,19 @@
 # frozen_string_literal: true
 
-class Spree::UserConfirmationsController < Devise::ConfirmationsController
+class UserConfirmationsController < Devise::ConfirmationsController
   helper 'spree/base', 'spree/store'
 
   include Spree::Core::ControllerHelpers::Auth
   include Spree::Core::ControllerHelpers::Common
   include Spree::Core::ControllerHelpers::Order
   include Spree::Core::ControllerHelpers::Store
-  include SolidusStarterFrontend::Taxonomies
+  include Taxonomies
+
+  layout 'storefront'
 
```

Comment:
Here and in the other devise controllers I think a less repetitive solution could be to set the `parent_controller` to `StoreController` in the devise configuration, unless there are reasons not to do so of course